include/SoundManager.h

```diff
@@ -22,14 +31,40 @@ class SoundManager
 			for (auto& it : this->sound_cache) {
 				Mix_FreeChunk(it.second);
 			}
+
+			for (auto& it : this->music_cache) {
+				Mix_FreeMusic(it.second);
+			}
 		}
 		
 		SoundManager(SoundManager const&) = delete;
 		void operator=(SoundManager const&) = delete;
 
+		std::map<const char*, Mix_Music*> music_cache;
 		std::map<const char*, Mix_Chunk*> sound_cache;
 
-		Mix_Chunk* loadSound(const char* fileName);
-		static void playSound(GameInternal* game, SoundTypes sound);
+		Mix_Music* loadMusic(const char* fileName); //!< Loads music from a file (mp3)
+		//! \returns a pointer to Mix_Music, which is added to a map in the AssetManager
```

Comment:
loadMusic() and loadSound() were called through the addMusic() and addSoundEffect() in the AssetManager at time of writing this, little oversight on my part, will change + remove loading sound from engine to chickengame

Reply:
In that case, "which is added to a map in the AssetManager" is missleading, as this is what it will be used for (the see also tag below seems appropriate for this), but implies `loadMusic()` adds it to the asset manager